run-ccsp2-build

```diff
@@ -19,12 +19,14 @@ export METRICS_UTILITY_REPORT_RHN_LOGIN="test_login"
 export METRICS_UTILITY_REPORT_SKU="MCT3752MO"
 export METRICS_UTILITY_REPORT_SKU_DESCRIPTION="EX: Red Hat Ansible Automation Platform, Full Support (1 Managed Node, Dedicated, Monthly)"
 
-/usr/bin/time --format='\n\nmemory %MK\ncpu %P\ntime %es\n\n' \
+TIME=`which gtime && echo 'gtime' || echo 'time'`
```

Comment:
```suggestion
TIME=`which gtime && echo 'gtime' || echo '/usr/bin/time'`
```

(all of them)

(because otherwise, bash will use its own builtin, which doesn't know --format either)

Reply:
Good catch